test-pcapplusplus.sh

```diff
@@ -22,7 +22,7 @@ if [ "$1" == "build_and_run" ]; then
     fi
 
     # Clone, configure, build and test PcapPlusPlus
-    git clone http://github.com/seladb/PcapPlusPlus.git
+    git clone -b fix-error-in-gcc15 http://github.com/seladb/PcapPlusPlus.git
```

Comment:
I'll revert this change before I merge this PR and after https://github.com/seladb/PcapPlusPlus/pull/1779 is merged

Reply:
Done in this commit: https://github.com/seladb/PcapPlusPlus-DockerImages/pull/52/commits/a24cbeaa97e98c09c580824ed83b88a59bef4404